src/content/structured/get-started/testing-with-cypress.mdx

```diff
@@ -0,0 +1,166 @@
+---
+path: "/get-started/test-components/testing-with-cypress"
+
+date: "2023-02-22"
+
+title: "Testing with Cypress"
+
+subTitle: "How to test our components with Cypress."
+
+navPriority: 4
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/testing-with-cypress.mdx"
+---
+
+## About Cypress
+
+[Cypress](https://www.cypress.io/) is a developer-friendly, Javascript-based front-end testing tool. It comes bundled with popular libraries like Mocha and Chai, allowing swift adoption. Advantages include snapshots at the time of test execution, allowing the user to hover specific commands to help debug and interrogate tests, implicit wait commands in test scripts, multiple browsers available to utilise in testing and real-time command execution with visual feedback.
+
+Cypress allows for End to End testing which renders a complete website, and Component testing which renders components on a development server.
+
+## Cypress and the Shadow DOM
+
+Cypress gives two ways in which to test `@ukic` components in the [shadow DOM](https://design.sis.gov.uk/get-started/install-components/testing-with-jest#what-is-the-shadow-dom). Firstly through calling a method `.shadow()` in the chain of assertions:
+
+```js
+it("has an app bar with title link", () => {
+  cy.get("ic-top-navigation", { includeShadowDom: true })
+    .shadow()
+    .find(".title-link")
+    .should("have.attr", "href", "/")
+    .should("have.text", "My App Title");
+});
+```
+
+The suffix should be attached to the parent element that contains the Shadow tree, after which you can search for the element you wish to test.
```

Comment:
be more like an instruction:
eg 
```suggestion
Attach the suffix to the parent element that contains the shadow tree.
```

not sure if the "after which....." is really needed